external/fv3fit/tests/emulation/models/test_microphysics_config.py

```diff
@@ -61,105 +61,17 @@ def test_Config_asdict():
     assert result == original
 
 
-def test_Config__processed_inputs():
```

Comment:
these tested the helper functions inside of `.build` and were easy to break as I changed their APIs. The coverage is the same after deleting them.